packages/cms-cli/commands/customObject/create.js

```diff
@@ -0,0 +1,57 @@
+const {
+  loadConfig,
+  validateConfig,
+  checkAndWarnGitInclusion,
+} = require('@hubspot/cms-lib');
+const { logger } = require('@hubspot/cms-lib/logger');
+const { logErrorInstance } = require('@hubspot/cms-lib/errorHandlers');
+const { getAbsoluteFilePath } = require('@hubspot/cms-lib/path');
+const { validatePortal, isFileValidJSON } = require('../../lib/validation');
+const { trackCommandUsage } = require('../../lib/usageTracking');
+const { setLogLevel, getPortalId } = require('../../lib/commonOpts');
+const { logDebugInfo } = require('../../lib/debugInfo');
+const { createObject } = require('@hubspot/cms-lib/api/customObject');
+
+exports.command = 'create <objectTypeId> <definition>';
+exports.describe = 'Create a custom object instance';
+
+exports.handler = async options => {
+  const { definition, objectTypeId } = options;
+  setLogLevel(options);
+  logDebugInfo(options);
+  const { config: configPath } = options;
+  loadConfig(configPath);
+  checkAndWarnGitInclusion();
+
+  if (!(validateConfig() && (await validatePortal(options)))) {
+    process.exit(1);
+  }
+  const portalId = getPortalId(options);
+
+  trackCommandUsage('custom-object-create', null, portalId);
+
+  const filePath = getAbsoluteFilePath(definition);
```

Comment:
I think ideally we'd probably want the option for an interaction key/value input, this was just to help us for now

Reply:
removed in favor of always doing batch for now